src/AccessKey.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.IdentityModel.Tokens;
+using Newtonsoft.Json;
+
+namespace Laserfiche.Oauth.Api.Client
+{
+    public class AccessKey
+    {
+        [JsonProperty("customerId")]
+        public string CustomerId { set; get; }
+
+        [JsonProperty("domain")]
+        public string Domain { set; get; }
+
+        [JsonProperty("client_id")]
```

Comment:
should be clientId and below should be jwk

Reply:
Oh, okay. I thought order [doesn't matter](https://stackoverflow.com/questions/16870416/does-the-sequence-of-the-values-matter-in-a-json-object), but yeah, I guess if we can get the order right, why not.

Reply:
not the order, the JsonProperty names should be updated

Reply:
Oh, right right 🤪

Reply:
Fixed.